[Bug] Failed to build FE on Centos 6

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.


### Version

Doris Version: master
OS: Centos 6

### What's Wrong?

Failed to build FE.

The root cause is that the prebuilt protoc-gen-grpc-java binary uses glibc on Linux and the version of glibc which Centos 6 uses is too old.

```shell
[INFO] --- protoc-jar-maven-plugin:3.11.4:run (default) @ fe-core ---
[INFO] Resolving artifact: com.google.protobuf:protoc:3.21.12, platform: linux-x86_64
protoc-jar: executing: [/tmp/protoc2355881335107243884.exe, --version]
/tmp/protoc2355881335107243884.exe: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by /tmp/protoc2355881335107243884.exe)
[INFO] Protoc command: /tmp/protoc2355881335107243884.exe
[INFO] Input directories:
[INFO]     /Programs/doris/fe/fe-core/../../gensrc/proto
[INFO] Output targets:
[INFO]     java: /Programs/doris/fe/fe-core/target/generated-sources (add: main, clean: false, plugin: null, outputOptions: null)
[INFO]     grpc-java: /Programs/doris/fe/fe-core/target/generated-sources (add: main, clean: false, plugin: null, outputOptions: null)
[INFO] Resolving artifact: io.grpc:protoc-gen-grpc-java:1.30.0, platform: linux-x86_64
[INFO]     Processing (java): olap_file.proto
protoc-jar: executing: [/tmp/protoc2355881335107243884.exe, -I/Programs/doris/fe/fe-core/../../gensrc/proto, --java_out=/Programs/doris/fe/fe-core/target/generated-sources, /Programs/doris/fe/fe-core/../../gensrc/proto/olap_file.proto]
/tmp/protoc2355881335107243884.exe: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by /tmp/protoc2355881335107243884.exe)
[ERROR] /Programs/doris/fe/fe-core/../../gensrc/proto/olap_file.proto [0:0]: /tmp/protoc2355881335107243884.exe: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by /tmp/protoc2355881335107243884.exe)
[ERROR] /Programs/doris/fe/fe-core/../../gensrc/proto/olap_file.proto [0:0]:
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Doris FE Project Parent POM 1.2-SNAPSHOT:
[INFO]
[INFO] Doris FE Project Parent POM ........................ SUCCESS [  3.597 s]
[INFO] fe-common .......................................... SUCCESS [ 28.691 s]
[INFO] spark-dpp .......................................... SUCCESS [ 17.857 s]
[INFO] fe-core ............................................ FAILURE [01:23 min]
[INFO] hive-udf ........................................... SKIPPED
[INFO] java-udf ........................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:13 min
[INFO] Finished at: 2023-03-09T12:00:21+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.os72:protoc-jar-maven-plugin:3.11.4:run (default) on project fe-core: protoc-jar failed for /Programs/doris/fe/fe-core/../../gensrc/proto/olap_file.proto. Exit code 1 -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :fe-core

```

### What You Expected?

Build FE successfully.

### How to Reproduce?

```shell
$ ./build.sh --fe --clean
```

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/fe/fe-core/pom.xml
+++ b/fe/fe-core/pom.xml
@@ -61,6 +61,7 @@ under the License.
             </properties>
         </profile>
         <profile>
+            <id>protoc_rosetta</id>
             <activation>
                 <os>
                     <name>Mac OS X</name>
@@ -70,6 +71,17 @@ under the License.
             <properties>
                 <protoc.artifact>com.google.protobuf:protoc:${protobuf.version}:exe:osx-x86_64</protoc.artifact>
                 <grpc.java.artifact>io.grpc:protoc-gen-grpc-java:${grpc.version}:exe:osx-x86_64</grpc.java.artifact>
+            </properties>
+        </profile>
+        <profile>
+            <id>protoc_command</id>
+            <activation>
+                <file>
+                    <exists>${doris.thirdparty}/installed/bin/protoc</exists>
+                </file>
+            </activation>
+            <properties>
+                <protoc.command>${doris.thirdparty}/installed/bin/protoc</protoc.command>
             </properties>
         </profile>
     </profiles>
@@ -900,7 +912,7 @@ under the License.
                             <goal>run</goal>
                         </goals>
                         <configuration>
-                            <!-- <protocCommand>${doris.thirdparty}/installed/bin/protoc</protocCommand> -->
+                            <protocCommand>${protoc.command}</protocCommand>
                             <!--You can use following protocArtifact instead of protocCommand, so that you don't need to install protobuf tools-->
                             <protocArtifact>${protoc.artifact}</protocArtifact>
                             <protocVersion>${protobuf.version}</protocVersion>

--- a/fe/hive-udf/pom.xml
+++ b/fe/hive-udf/pom.xml
@@ -63,10 +63,6 @@ under the License.
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
-            <groupId>org.apache.hive</groupId>
-            <artifactId>hive-serde</artifactId>
-        </dependency>
         <!-- https://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient -->
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>

